utils/Colors.ts

```diff
@@ -1 +1,46 @@
-export const WMBR_GREEN = '#00843D';
+export const CORE_COLORS = {
+  WHITE: '#FFFFFF',
+  BLACK: '#000000',
+  GRAY_100: '#E0E0E0',
+  GRAY_200: '#CCCCCC',    
+  GRAY_300: '#BBBBBB',
+  GRAY_400: '#AAAAAA',
+  GRAY_500: '#999999',
+  GRAY_600: '#888888',
+  GRAY_700: '#555555',
+  GRAY_900: '#333333',
+  GRAY_950: '#2a2a2a',
+  GRAY_975: '#1a1a1a',
+  RED_500: '#FF4444',
+  GREEN_500: '#00D17A',
+  WMBR_GREEN: '#00843D',
+} as const;
+
+export const COLORS = {
+  TEXT: {
+    PRIMARY: CORE_COLORS.WHITE,           // #FFFFFF - Main headings, primary text (86 uses)
```

Comment:
I'd vote getting rid of at least the X uses part since that'll drift very quickly, but I also think worth omitting the whole comment - easy to hover/scroll up

Reply:
Yeah good call. Also happy to discuss this color tokenization strategy at our meeting later today -- I'm kind of winging it and don't want to go in this direction if there are other ideas.